Sources/AblyChat/Metadata.swift

```diff
@@ -1,2 +1,44 @@
 // TODO: (https://github.com/ably-labs/ably-chat-swift/issues/13): try to improve this type
-public typealias Metadata = [String: (any Sendable)?]
+
+import Foundation
+
+// Define protocol for types that are Sendable and Encodable
+public protocol SendableEncodable: Sendable, Codable, Hashable {}
```

Comment:
Is there a need for this? The name is particularly confusing because it doesn't mention `Hashable`.

Reply:
removed, was trying some stuff that didn't work out